src/test/java/runners/DashboardTestRunner.java

```diff
@@ -0,0 +1,6 @@
+package runners;
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+@CucumberOptions(features="src/test/resources/Features/DashboardTest.feature",glue={"StepDefinition"})
+public class DashboardTestRunner extends AbstractTestNGCucumberTests {
```

Comment:
Join the LoginTestRunner and DashboardTestRunner together like CucumberTestRunner.
You will be enough to provide the path to the all feature files and run them all, at least for now, and path to the steps.
Just an example:
https://www.way2automation.com/create-runner-file-in-cucumber-bdd/